aux/custom-first-boot.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Custom first boot script
+ConditionFileIsExecutable=/usr/local/sbin/custom-first-boot
+ConditionFirstBoot=yes
```

Comment:
I have found a better solution - there is a built-in firstboot support in systemd. So I am changing it to that:

https://www.freedesktop.org/software/systemd/man/systemd.unit

Additional Wants/Before/After were added below as recommended in the systemd man page.